### DISCUSSION TITLE
Flow Static Type Checking Support

### DISCUSSION THREAD

**Discussion by jnahumphreys** [storybookjs/storybook#16806] [Help]
Hi, 

Does anyone know if the current version of StoryBook supports [Flow Static Type checking](https://flow.org/), specifically auto generated docs table (instead of using PropTypes)?

Thanks in advance

**Comment by shilman** [comment]
I think it works for React although it's not officially supported. Storybook uses `react-docgen` under the hood and that supports flow

**Reply by jnahumphreys** [reply]
Thanks, that aligns with what I’ve understood from looking into it. 

I’ll run a short spike next week to see how it performs and report back on this thread for future reference.
